.eslintrc.js

```diff
@@ -33,6 +33,9 @@ module.exports = {
     'no-underscore-dangle': 'off',
     'react/require-default-props': 'off',
     'no-bitwise': 'off',
+    'no-console': 'off',
+    '@typescript-eslint/no-explicit-any': 'off',
+    'no-await-in-loop': 'off',
```

Comment:
We should keep these 3 rules on. 

- `no-console`: Check if it's possible to only disallow the `console.log` but keep allowing the .error and the .warning ones. 
- `no-explic-any`: Lets keep this one as well, let's replace the `any` that we have in the code with `unknown`
- `no-await-in-loop`: In the rare cases where we REALLY should have an await inside a for loop, let's disable it with a comment.